format/FormatHDF5.py

```diff
@@ -8,6 +8,10 @@
 
 
 class FormatHDF5(FormatMultiImage, Format):
+    @classmethod
+    def is_abstract(cls):
```

Comment:
`is_abstract()` -> `abstract_format()` may be more in keeping? Proposal, not request